config/supervisor.conf

```diff
@@ -0,0 +1,13 @@
+[program:app-uwsgi]
+command = /usr/local/bin/uwsgi --ini /code/config/uwsgi.ini
+
+[program:nginx-app]
+command = /usr/sbin/nginx
+
+[program:app-setup]
+command = /bin/bash /code/config/setup_app.sh
+
+[program:cron]
```

Comment:
Honestly, I have no experience with `supervisord`. I think I got some idea from https://docs.docker.com/config/containers/multi-service_container/ in context of docker. But I have no clue as to what we are trying to achieve with this block here?

Reply:
It creates a file with env variables (which the Cron Jobs can access) and then starts the cron jobs.

Reply:
Okay.

Reply:
Okay.